src/main/java/catan/services/util/play/PlayUtil.java

```diff
@@ -40,6 +40,12 @@ public void updateNextMove(GameBean game) throws GameException {
         }
     }
 
+    public void resetDices (GameBean game) {
```

Comment:
was it agreed that we have to reset them when user ends his turn? I think we don't handle it on front-end